recipes/elasticsearch.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/* (c) Malte Blaettermann <malte.blaettermann@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+set('elasticsearch', [
+    'host' => null,
+    'protocol' => 'https',
+    'index' => 'deployments',
+    'index-date-pattern' => null,
+    'doc_type' => 'deployment',
+    'username' => null,
+    'passoword' => null,
+]);
+
+env('elasticsearch_basic_auth', function() {
+    $config = get('elasticsearch');
+    $basicAuth = '';
+    if ($config['username'] && $config['password'] && $config['protocol'] === 'https') {
+        $basicAuth = sprint(' --user name:password %s:%s', $config['username'], $config['password']);
+    }
+
+    return $basicAuth;
+});
+
+env('elasticsearch_index_name', function () {
+    $config = get('elasticsearch');
+    $indexName = $config['index'];
+    if(is_string($config['index-date-pattern'])) {
+        $indexName .= '-' . date($config['index-date-pattern']);
+    }
+
+    return $indexName;
+});
+
+env('elasticsearch_get_revision', function() {
+    return runLocally(env('bin/git'). ' rev-parse --short HEAD');
+});
+
+env('elasticsearch_cmd', function () {
+    $config = get('elasticsearch');
+    if (!is_string($config['host'])) {
+        throw new \RuntimeException('Please set an elasticsearch host via set(\'elasticsearch.host\', \'your.host:port\' )');
+    }
+
+    $payload = [
+        'revision' => env('elasticsearch_get_revision'),
+        'stage' => env('stage'),
+        'datetime' => date(DateTime::ISO8601),
+    ];
+
+    $basicQueryString = sprintf('curl %s -XPOST %s://%s/%s/%s/  -d "%s"',
+        env('elasticsearch_basic_auth'),
+        $config['protocol'],
+        $config['host'],
+        $config['host'],
+        env(elasticsearch_index_name()),
+        $config['doc_type'],
+        json_encode($payload)
+        );
```

Comment:
What would be best to fix it? Is deployer following a specific CS ? Or do just talk about that space in front of `json_encode`?